nvtabular/io.py

```diff
@@ -729,6 +729,9 @@ def __init__(self, paths, part_size, storage_options=None):
     def to_ddf(self, columns=None):
         raise NotImplementedError(""" Return a dask_cudf.DataFrame """)
 
+    def make_iter(self, columns=None, indices=None):
```

Comment:
Unless I am missing something, you shouldn't need the `make_iter` definitions anymore.  This logic was recently moved up to the Dataset level.

Reply:
removed